portfolio/src/main/webapp/script.js

```diff
@@ -116,3 +118,72 @@ function deleteComments() {
   let request = new Request('/delete-data', {method:'DELETE'});
   fetch(request).then(unused => {/*console.log('Delete finishes'); DEBUG Tool*/loadComments()});
 }
+
+function addPageButtons(max) {
+  numberOfComments = max;
+  numberOfPages = Math.ceil(numberOfComments / 5);
+  const pageNumbers = document.getElementById('pageNumbers');
+  pageNumbers.innerHTML = '';
+  let buttonText = ['<', '1'];
+  for(let i=2; i <= numberOfPages; i++){
```

Comment:
Could you clean up the spaces in this for-loop and the one below? There should be spaces around the "=" operator and all comparison operators.